cookbooks/aws-parallelcluster-environment/files/ubuntu/network_interfaces/configure_nw_interface.sh

```diff
@@ -41,8 +41,8 @@ if [ "${STATIC_IP_CONFIG}" = "" ]
 fi
 
 FILE="/etc/netplan/${DEVICE_NAME}.yaml"
-SUFFIX=$(printf "%03d" $NETWORK_CARD_INDEX)$(printf "%03d" $DEVICE_NUMBER)
-ROUTE_TABLE="1${SUFFIX}"
+SUFFIX=$(printf "%03d" $NETWORK_CARD_INDEX)$(printf "%02d" $DEVICE_NUMBER)
+ROUTE_TABLE="$(( $SUFFIX + 10 ))"
```

Comment:
I would have expected a change in the kitchen tests after this change.

Reply:
Could you please execute [kitchen tests](https://github.com/aws/aws-parallelcluster-cookbook/blob/develop/cookbooks/aws-parallelcluster-environment/kitchen.environment-config.yml#L329), verify if they are still correct and eventually improve coverage ? Thanks!